tutorials/io/70_reading_eyetracking_data.py

```diff
@@ -78,29 +78,43 @@
 new line, the y-coordinate *increased*, which is why the ``ypos_right`` channel
 in the plot below increases over time (for example, at about 4-seconds, and
 at about 8-seconds).
+
+.. seealso::
+
+    :ref:`tut-eyetrack`
 """
 
 # %%
-from mne.datasets import misc
-from mne.io import read_raw_eyelink
+import mne
 
 # %%
-fpath = misc.data_path() / "eyetracking" / "eyelink"
-raw = read_raw_eyelink(fpath / "px_textpage_ws.asc", create_annotations=["blinks"])
-custom_scalings = dict(eyegaze=1e3)
-raw.pick(picks="eyetrack").plot(scalings=custom_scalings)
+fpath = mne.datasets.misc.data_path() / "eyetracking" / "eyelink"
+fname = fpath / "px_textpage_ws.asc"
+raw = mne.io.read_raw_eyelink(fname, create_annotations=["blinks"])
+cal = mne.preprocessing.eyetracking.read_eyelink_calibration(
+    fname,
+    screen_distance=0.7,
+    screen_size=(0.53, 0.3),
+    screen_resolution=(1920, 1080),
+)[0]
+mne.preprocessing.eyetracking.convert_units(raw, calibration=cal, to="radians")
 
+# %%
+# Visualizing the data
+# ^^^^^^^^^^^^^^^^^^^^
 
 # %%
-# .. important:: The (0, 0) pixel coordinates are at the top-left of the
-#               trackable area of the screen. Gaze towards lower areas of the
-#               the screen will yield a relatively higher y-coordinate.
-#
-# Note that we passed a custom `dict` to the ``'scalings'`` argument of
-# `mne.io.Raw.plot`. This is because MNE's default plot scalings for eye
-# position data are calibrated for HREF data, which are stored in radians
-# (read below).
+cal.plot()
 
+# %%
+custom_scalings = dict(pupil=1e3)
+raw.pick(picks="eyetrack").plot(scalings=custom_scalings)
```

Comment:
This ends up a bit wacky:

![image](https://github.com/user-attachments/assets/16ca6711-998f-43eb-87ba-ce925888684e)

If you set the `units` and `scalings` it will probably be more reasonable I think

Reply:
Sorry @larsoner Im not sure what you mean by setting the `units` and if you are referring to the eyegaze channel or pupil channel (I assume eyegaze) - Can you clarify?

Reply:
I meant for `plot_raw` but I see now that we only have it for other viz functions like [`plot_evoked`](https://mne.tools/dev/generated/mne.viz.plot_evoked.html), so never mind! Also not sure it would actually help...